WorkloadManagementSystem/Service/JobMonitoringHandler.py

```diff
@@ -31,16 +33,28 @@
 def initializeJobMonitoringHandler(serviceInfo):
 
   global gJobDB, gJobLoggingDB, gTaskQueueDB
+
   gJobDB = JobDB()
   gJobLoggingDB = JobLoggingDB()
   gTaskQueueDB = TaskQueueDB()
+
   return S_OK()
 
 
 class JobMonitoringHandler(RequestHandler):
 
   def initialize(self):
 
+    global gJobElasticDB, gJobDB
```

Comment:
This can be included in line 35.

Reply:
I think we need to reference the variables here as we are modifying the global values in this function. Line 35 is in a different function.